ui/apps/platform/src/Containers/VulnMgmt/List/Cves/ListCVEs.utils.js

```diff
@@ -27,16 +25,15 @@ export function getFilteredCVEColumns(columns, workflowState, isFeatureFlagEnabl
 
     // No need to show entities in the node component or cluster context.
     const shouldKeepEntitiesColumn =
-        !workflowState.isPrecedingSingle(entityTypes.COMPONENT) ||
+        !workflowState.isPrecedingSingle(entityTypes.NODE_COMPONENT) ||
         !workflowState.getSingleAncestorOfType(entityTypes.NODE);
     // special case CLUSTER CVE under CLUSTER
     const clusterCveUnderCluster =
         workflowState.getSingleAncestorOfType(entityTypes.CLUSTER) &&
         currentEntityType === entityTypes.CLUSTER_CVE;
 
     // TODO: remove this temporary conditional check, after generic CVE list is removed
-    const shouldKeepCveType =
-        currentEntityType === entityTypes.CVE || currentEntityType === entityTypes.CLUSTER_CVE;
+    const shouldKeepCveType = currentEntityType === entityTypes.CLUSTER_CVE;
```

Comment:
Does TODO comment mean do keep `shouldKeepCveType` pardon pun, only delete CVE entity type?

Reply:
@vjwilson when I started to delete the comment, I noticed related comment a few lines below it:

```js
    return columns.filter((col) => {
        switch (col.accessor) {
            // TODO: remove after generic CVE list is removed
            case 'vulnerabilityTypes': {
                return !!shouldKeepCveType;
            }
```

Reply:
Ah, that comment is inaccurate. The comment should be deleted but the condition still applies.

(We are using the same CVE table for all 3 Postgres types of CVEs, but with the appropriate columns filtered out based on type. The column list starts with all possible columns, and `vulnerabiltyTypes` column is only left when `shouldKeeepCveType` is `true`.)